admin-dev/themes/new-theme/js/pages/product/services/paginated-combinations-service.ts

```diff
@@ -68,6 +76,21 @@ export default class PaginatedCombinationsService implements PaginationServiceTy
     return $.get(this.router.generate('admin_products_combinations', requestParams));
   }
 
+  //@todo: duplicate from combination-service, except that passes filters
```

Comment:
Then it means we should probably remove the old one from the combination-service  and only use this one no?

Reply:
I believe the edition modal also uses this to get its list and allow navigation But it actually seems better that the navigation for the modal uses the same IDs as the ones filtered This way you can filter red t-shirt (for example) and only edit the red ones one by one via the modal

Reply:
I injected PaginatedCombinationsService into CombinationModal now, but it still ignores filters :man_shrugging: